tests/Aspire.Hosting.Azure.Tests/AzureStorageEmulatorFunctionalTests.cs

```diff
@@ -135,9 +135,10 @@ public async Task VerifyAzureStorageEmulatorResource()
 
     [Fact]
     [RequiresDocker]
-    [QuarantinedTest("https://github.com/dotnet/aspire/issues/9139")]
```

Comment:
Are we sure that this can be dropped? For quarantined tests we want to take it out after it has been green for a certain number of runs (~100 right now).

Reply:
https://github.com/dotnet/aspire/pull/9364

Reply:
Thank you!